conjure-core/src/test/resources/example-external-types.yml

```diff
@@ -8,6 +8,11 @@ types:
       base-type: string
       external:
         java: com.palantir.example.ExternalIntegerImport
+    ExampleLong:
+      base-type: string
+      external:
+        java: java.lang.Long
```

Comment:
To be super safe, we may want to add validation that using `base-type: bearertoken` cannot be marked safe/unsafe because it's do-not-log.

Reply:
added some validation logic for that!